test/e2e/framework/util.go

```diff
@@ -764,28 +764,28 @@ func waitForServiceAccountInNamespace(c clientset.Interface, ns, serviceAccountN
 }
 
 func WaitForPodCondition(c clientset.Interface, ns, podName, desc string, timeout time.Duration, condition podCondition) error {
-	Logf("Waiting up to %[1]v for pod %[2]s status to be %[3]s", timeout, podName, desc)
+	Logf("Waiting up to %v for pod %q in namespace %q to be %q", timeout, podName, ns, desc)
```

Comment:
I think no because leaving "status" in this message is misleading since the insert used is the passed-in `desc`. The passed in `condition` func may use the pod's Status to decide when to stop the loop but we don't know that here.